app/src/main/java/ch/epfl/sdp/contamination/fragment/UserInfectionFragment.java

```diff
@@ -235,25 +229,15 @@ private void executeHealthStatusChange() {
     }
 
     private void sendRecoveryToFirebase() {
-        // TODO: @Lucie I think that doesn't upload anything to Firestore
+        // TODO: I think that doesn't upload anything to Firestore
+        // --->> Are you sure? From what I read on the doc,
+        //                      update method from DocumentReference updates on Firebase
         // TODO: [LOG]
         Log.e("RECOVERY_SENDER", account.getId());
         DocumentReference ref = documentReference(privateUserFolder, account.getId());
         ref.update(privateRecoveryCounter, FieldValue.increment(1));
     }
 
-    private CompletableFuture<Boolean> retrieveUserInfectionStatus() {
```

Comment:
The user status is stored locally now that's correct ?

Reply:
Also. Both locally and remotely

Reply:
But why do we need this method?
The status should be taken from the Carrier initialized by LocationService